terra-jupyter-python/Dockerfile

```diff
@@ -42,15 +42,19 @@ RUN pip3 -V \
 
 RUN pip3 install --upgrade markupsafe==2.1.2
 
-ENV USER jupyter
+# Turn off beatrix-jupyterlab and bq_stats plugins, they don't activate cleanly in our version
+# of jupyter and they're google-managed packages we don't have access to older versions of.
+# Retain two plugins: google.cloud.bigquery and sql
+RUN sed -i "s/^\( *'beatrix_jupyterlab',\)/#\1/g" /etc/ipython/ipython_kernel_config.py
+RUN sed -i "s/^\( *'bq_stats',\)/#\1/g" /etc/ipython/ipython_kernel_config.py
```

Comment:
A reasonable compromise, that speaks to the need to delegate package selection to users.